__init__.py

```diff
@@ -0,0 +1,73 @@
+from openfecwebapp.local_config import port, debug, host
+from flask import Flask, render_template, request
+from openfecwebapp.views import (render_search_results, render_table,
+    render_page)
+from openfecwebapp.api_caller import (load_search_results,
+    load_single_type, load_totals)
+
+app = Flask(__name__)
+
+def _filter_empty_params(params):
+    new_params = {}
+    for p in params:
+        if params[p]:
+            new_params[p] = params[p]
+
+    return new_params
+
+def _add_fields_star(params):
+    # move from immutablemultidict -> multidict -> dict
```

Comment:
Are you sure you *want* a mutable dict? If you want this to be a docstring, it needs to be a string rather than a pound.